agent/charms/testflinger-agent-host-charm/tests/integration/test_charm_integration.py

```diff
@@ -0,0 +1,29 @@
+from pathlib import Path
+import pytest
+from pytest_operator.plugin import OpsTest
+
+# Root of the charm we need to build is two dirs up
+CHARM_PATH = Path(__file__).parent.parent.parent
+APP_NAME = "testflinger-agent-host"
+
+
+#@pytest.mark.skip_if_deployed
+#@pytest.mark.setup
```

Comment:
is there a need for these commented out decorators?

Reply:
fixed, thanks!